Tests/ValetIntegrationTests/SecureEnclaveIntegrationTests.swift

```diff
@@ -199,4 +202,106 @@ class SecureEnclaveIntegrationTests: XCTestCase
             XCTAssertEqual(try otherValet.string(forKey: key), value)
         }
     }
+
+    // MARK: string(forKey:withPrompt:)
+
+    func test_stringForKeyWithPrompt_throwsItemNotFoundForKeyWithNoValue() throws {
+        XCTAssertThrowsError(try valet.string(forKey: key, withPrompt: prompt)) { error in
+            XCTAssertEqual(error as? KeychainError, .itemNotFound)
+        }
+    }
+
+    func test_stringForKeyWithPrompt_retrievesStringForValidKey() throws {
+        try valet.setString(passcode, forKey: key)
+        XCTAssertEqual(passcode, try valet.string(forKey: key, withPrompt: prompt))
+    }
```

Comment:
Welp. I'm surprised I missed these test cases before! Thanks for adding them.

(I haven't validated that these didn't exist already but I believe you that they didn't!)